app/views/course/provider.njk

```diff
@@ -5,20 +5,27 @@
 {% block pageNavigation %}{{ govukBackLink({ href: paths.back }) }}{% endblock %}
 
 {% block primary %}
-  {# Paragraph font size class suffixes: -xl, -l or -m #}
-  <div class="govuk-!-margin-top-4">
-  {{ govukInput({
-    label: {
-      text: 'Provider name'
-    },
-    autocomplete: "name",
-    id: "candidate-name",
-    name: "candidate[name]",
-    value: data["candidate"]["name"]
-  }) }}
+  <div class="govuk-form-group">
+    <label for="courses-{{ courseId }}-provider" class="govuk-label">Provider name</label>
+    <div id="provider-autocomplete-containter"></div>
```

Comment:
I take it you chose to use an empty `div` rather than enhance a `select` due to the amount of data involved? I know it’s only a prototype at this stage, but what would the non-JS fallback be if this design were to make it into the live service? Just enhance an `input type="text"` and use server-side validation?

Reply:
We can probably enhance a text input in the way that Find does it.

Reply:
Maybe we should do that here then (or as close as we can get to that). Just wary of inaccessible code slipping into the prototype (and the possibility of this then causing issues in usability sessions).